lib/ips/metadata.rb

```diff
@@ -4,51 +4,92 @@ module IPS
   class Metadata < Inferno::TestKit
     id :ips_test_kit
     title 'International Patient Summary Test Kit'
+    version VERSION
+    last_updated LAST_UPDATED
+
+    # Configuration
+    suite_ids [:ips]
+    tags ['IPS']
+    maturity 'Low'
+    authors ['MITRE Inferno Team']
+    repo 'https://github.com/inferno-framework/ips-test-kit'
+
+    # External Links
+    links [
+      {
+        label: 'Implementation Guide',
+        url: 'https://hl7.org/fhir/uv/ips/STU1.1/'
+      },
+      {
+        label: 'Source Code',
+        url: 'https://github.com/inferno-framework/ips-test-kit'
+      },
+      {
+        label: 'Issues',
+        url: 'https://github.com/inferno-framework/ips-test-kit/issues'
+      },
+      {
+        label: 'Inferno Framework',
+        url: 'https://inferno-framework.github.io/inferno-core/'
+      }
+    ]
+
```

Comment:
This isn't needed at the test kit metadata level (which, yeah, it probably should but for historic reasons we have it at the test suite level).  And in fact, this breaks inferno so that it doesn't load.  So you have to delete it.